backend/curriculum_tracking/management/commands/import_curriculum.py

```diff
@@ -0,0 +1,74 @@
+from core.models import Curriculum
+from curriculum_tracking.models import (
+    CurriculumContentRequirement,
+    ContentItemOrder,
+    ContentItem,
+)
+from curriculum_tracking.card_generation_helpers import get_ordered_content_items
+import json
+from django.core.management.base import BaseCommand
+import pandas as pd
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        json_file = "dev_helpers/data/intro-to-tilde-course.json"
+        save_curriculum_to_db(json_file)
+
+
+def save_curriculum_to_db(json_file):
+    with open(json_file) as f:
+        data = json.load(f)
+
+    df_dict = {i: pd.DataFrame(data[i]) if i != 'curriculum' else pd.DataFrame(pd.Series(data[i])).transpose() for i in data}
```

Comment:
I think that using pandas here makes things more complicated than is needed. `data` is really just a dictionary. The data isn't big or anything.  I think it would be very good for you to be comfortable using foundational python. Python is very powerful even when nt using pandas.